webapp/www/templates/client_setup.html

```diff
@@ -57,14 +59,16 @@ <h4>Install the {{base_app}} app</h4>
           <h4>Join this study</h4>
         </div>
         <div id="downloadRow">
-          <a id="groupLink" href="emission://change_client?new_client={{client_label}}&clear_local_storage={{clear_local_storage_flag}}&clear_usercache={{clear_usercache_flag}}">
+          <a id="groupLink" href="emission://change_client?new_client={{greentrip}}&clear_local_storage={{clear_local_storage_flag}}&clear_usercache={{clear_usercache_flag}}">
```

Comment:
I am completely confused now. Aren't you going to randomly assign people to either greentrip or control group?
If you hardcode greentrip here, how is your recruitment supposed to happen?

Reply:
It's hardcoded to greentrip because I don't have the other qr code. The generated qr code through angular was not working so now I'm going to have it randomly choose from the two qr code images to display.

Reply:
as I said in slack, you can get the other QR code by using `new_client=urap2017control` instead of `new_client=greentrip`.

Reply:
Is this supposed to be resolved? Because I still don't see any kind of randomness here.

Reply:
The code originally without the images of the qr codes presented nothing with no error so there was no way I could debug. It seemed as though the qr code generator was just not working period, which is why I switched to images.

Reply:
I have no objection with switching to images, but you need to randomly switch between two images, right?

Reply:
where is the `Math.random` that chooses between the images? which line number?

Reply:
The qr code generation is not working. Changing this did not help.